bedrock/mozorg/templates/mozorg/home/home-quantum.html

```diff
@@ -32,6 +32,71 @@
 
 {% block site_header %}
 {% include 'mozorg/home/includes/home-icons-sprite.svg' %}
+  {% if l10n_has_tag('home-fundraiser-EOY2017') %}
+    <aside class="promo" id="fundraiser">
+      <h1>
+        {{ _('We all love the web. Join Mozilla in defending it.') }}
+      </h1>
+
+      <p>
+      {% if LANG == 'en-US' %}
+        Let’s protect the world’s largest public resource for future
+        generations. A few times a year, Mozilla asks for donations. Just
+        chipping in $3 can help us keep the web healthy, wonderful and
+        welcoming to all.
+      {% else %}
+        {#
+          L10n: The "cup of coffee" item below is meant to represent an
+          inexpensive item that an average person would purchase daily/without
+          hesitation.
+        #}
+        {% trans %}
+          Let’s protect the world’s largest public resource for future
+          generations. A few times a year, Mozilla asks for donations. Just
+          chipping in the price of a cup of coffee can help us keep the web
+          healthy, wonderful and welcoming to all.
+        {% endtrans %}
+      {% endif %}
+      </p>
+
+      <form id="fundraiser-form" method="get" action="https://donate.mozilla.org/{{ LANG }}/">
+        <fieldset class="donate-options">
+          {% for amount in donate_params.preset_list %}
+            <label for="donate{{ amount }}" {% if amount == donate_params.default %}class="selected"{% endif %}>
+              <input type="radio" value="{{ amount }}" id="donate{{ amount }}" name="amount"
+              {% if amount == donate_params.default %}checked="checked"{% endif %} />
+              {# L10n: Inserts a sum in US dollars, e.g. '$100'. Adapt the string in your translation for your locale conventions if needed, ex: %(sum)s US$ #}
+              {{ _('$%(sum)s')|format(sum=amount) }}
+            </label>
+          {% endfor %}
+
+          <label for="donate-custom" class="own-amount">{{ _('Enter your own amount') }}</label>
+          <input type="text" name="amount" id="donate-custom" />
+        </fieldset>
+        <fieldset class="recurring">
+          <label for="monthly">
+            <input type="radio" value="monthly" id="monthly" name="frequency"> {{ _('Monthly') }}
+          </label>
+
+          <label for="onetime">
+            <input type="radio" value="single" id="onetime" name="frequency" checked> {{ _('One-time') }}
+          </label>
+
+          <input type="hidden" name="currency" value="{{ donate_params.currency }}" />
+          <input type="hidden" name="presets" value="{{ donate_params.presets }}" />
+
+          {# GA params #}
+          <input type="hidden" name="ref" value="EOYFR2015" />
+          <input type="hidden" name="utm_campaign" value="EOYFR2015" />
```

Comment:
I know it’s a WIP, but heads up that we will want an updated tag here. `EOYFR2017` should be okay, unless @vojtechmzl would prefer something else